test/exe/Main.hs

```diff
@@ -2274,25 +2276,25 @@ pluginSimpleTests =
           )
       ]
 
-pluginParsedResultTests :: TestTree 
```

Comment:
Worth linting about trailing whitespace on lines? Such additional noise in diffs is a pain.

Reply:
I had to install the vscode plugin that make it honour the already existing .editorconfig, it removed the existing trailing whitespace

Reply:
Yeah, super annoying VS Code doesn't do that by default :(